scripts/game_structure/load_cat.py

```diff
@@ -490,7 +529,7 @@ def csv_load(all_cats):
             for app_id in inter_cat.apprentice:
                 app = Cat.all_cats.get(app_id)
                 # Make sure if cat isn't an apprentice, they're a former apprentice
-                if "apprentice" in app.status:
+                if CatRank.APPRENTICE == app.status.rank:
```

Comment:
```suggestion
                if app.status.rank == CatRank.APPRENTICE:
``` 
Minor thing but the changing value should always be first